recipes/libosxunwind/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "libosxunwind" %}
+{% set version = "0.0.3" %}
+{% set sha256 = "862d060d4361e5a5c5b0d02508a114a51aa66b5bb25462b2bf3a9f48d420a54d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/JuliaLang/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not osx]
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  commands:
+    - test -f "${PREFIX}/include/libunwind.h"
+    - test -f "${PREFIX}/include/unwind.h"
+    - test -d "${PREFIX}/include/mach-o"
+    - test -f "${PREFIX}/lib/libosxunwind.a"
+    - test -f "${PREFIX}/lib/libosxunwind.dylib"
+
+    - conda inspect linkages -p "${PREFIX}" "{{ name }}"
+    - conda inspect objects -p "${PREFIX}" "{{ name }}"
+
+about:
+  home: https://github.com/JuliaLang/libosxunwind
+  license: APSL-2.0
+  # No license file included. Raise an issue upstream.
+  #
+  # xref: https://github.com/JuliaLang/libosxunwind/issues/10
+  license_file: "{{ RECIPE_DIR }}/LICENSE.txt"
+  license_url: https://opensource.apple.com/apsl
```

Comment:
No license file included upstream. However this was taken from Apple's source code and all of the file's headers note they are under the Apple Public Source License 2.0. So have added the license file from Apple in the recipe/package and raised issue ( https://github.com/JuliaLang/libosxunwind/issues/10 ) upstream for clarification and a license addition.